src/components/about-us/section-05/index.js

```diff
@@ -404,62 +423,64 @@ export default class Section5 extends PureComponent {
+        <Container>
+          <SectionWrapper>
+            <Title><span>大事紀</span></Title>
+            <Circle color={`${colors.black}`}>
+              <Circle color={`${colors.gray.gray96}`}/>
+              <Rect color={`${colors.secondaryColor}`}/>
+            </Circle>
+            <Content>
+              <AccordionTimeline>
                 <List
                   unfoldArray={this.state.unfoldArray}
                   sortedData={sortedData}
                   sortedDataGroupByYear={sortedDataGroupByYear}
                   foldAndUnfold={this._foldAndUnfold}
                   getYearContentHeight={this._getYearContentHeight}  
                 />
-              </Timeline>
-            </RunningTimeline>
-            <YearTag>
-              <p>{this.state.currentYear}</p>
-            </YearTag>
-          </Content>
-        </SectionWrapper>
-        <BorderBottom 
-          fixed={this.state.isBorderBottomfixed}
-          zIndex={this._getBorderZIndex()}
-        />
-        <Waypoint
-          onPositionChange={({ previousPosition, currentPosition }) => this._onPositionChange(previousPosition, currentPosition)}
-          fireOnRapidScroll
-        />
-      </Container>
+              </AccordionTimeline>
+              <YearRange>
+                <p><span>{yearList[yearList.length - 1]}</span></p>
+                <p>
+                  <span>{yearList[0]}</span>
+                  <img src={arrow} />
+                </p>
+              </YearRange>
+              <RunningTimeline>
+                <Timeline
+                  ref={scrollingContent => this.scrollingContent = scrollingContent}
+                  childrenHeight={this.state.timelineScrollingHeight * timelineScrollingPortion} 
+                  autoScrolling={this.state.timelineScrolling}
+                  startAutoScroll={this._startTimelineAutoScrolling}
+                  stopAutoScroll={this._stopTimelineAutoScrolling}
+                  yearContentHeight={this.state.yearContentHeight}
+                  getYear={this._getYear}>
+                  <List
+                    unfoldArray={this.state.unfoldArray}
+                    sortedData={sortedData}
+                    sortedDataGroupByYear={sortedDataGroupByYear}
+                    foldAndUnfold={this._foldAndUnfold}
+                    getYearContentHeight={this._getYearContentHeight}  
+                  />
+                </Timeline>
+              </RunningTimeline>
+              <YearTag>
+                <p>{this.state.currentYear}</p>
+              </YearTag>
+            </Content>
+          </SectionWrapper>
+          <BorderBottom 
+            fixed={this.state.isBorderBottomfixed}
+            zIndex={this._getBorderZIndex()}
+          />
+          <Waypoint
+            onPositionChange={({ previousPosition, currentPosition }) => this._onPositionChange(previousPosition, currentPosition)}
+            fireOnRapidScroll
+          />
+        </Container>
+      </Border>
```

Comment:
All those `Border`s in `about-us/section-0[\d]/index.js` should be moved into `about-us/index.js`.
Hence, you just need to declare `Border` one time.